CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+## CHANGELOG
+
+### 1.2.0 (2020-07-21)
+**SDK**
+- Upgraded build setup and some internal dependencies
+- Added support for customization of user agent information
+- Added feature to support backward & forward navigation support in a mini app
```

Comment:
This also needs more explanation. Maybe just a link to the correct section in the user guide? Actually, I don't see this in the user guide, so maybe that needs to be added?

Reply:
The user guide is updated now.